README.md

```diff
@@ -1 +1,120 @@
-# capstone-project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone Project 1] <a name="about-project"></a>
+
+This project is the capstone project that portrays a conference website.
+
+**[Capstone Project]** is a simple project demonstrating responsive conference website.
+
+## 🚀 VIEW ONLINE
+
+<a name="<<https://king-benx.github.io/capstone-project-1/>>">https://king-benx.github.io/capstone-project-1/</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
```

Comment:
- [ ] According to the project requirement, it was stated that you record a `Loom video` where you give a presentation of what your project is all about. Kindly record the video and add the link to your README file.

> N.B:  ![loom](https://user-images.githubusercontent.com/78634841/210804861-8a26e53f-1ab8-4cbf-b42d-e0af9d4824bb.png)